tests/integration/test_design.py

```diff
@@ -1859,3 +1867,15 @@ def test_get_active_design(modeler: Modeler):
     design1 = modeler.create_design("Design1")
     active_design = modeler.get_active_design()
     assert active_design.design_id == design1.design_id
+
+
+def test_get_collision(modeler: Modeler):
+    """Test the get_collision method of the bodies."""
```

Comment:
```suggestion
    """Test the ``get_collision()`` method of the bodies."""
```
Descriptions should generally be in plain English. Should we rewrite to: "Test to validate getting the collision of the bodies."